test/functional/helpers.lua

```diff
@@ -290,6 +290,22 @@ local function expect(contents)
   return eq(dedent(contents), curbuf_contents())
 end
 
+local function rmdir(path)
+  if lfs.attributes(path, 'mode') ~= 'directory' then
+    return nil
+  end
+  for file in lfs.dir(path) do
+    if file == '.' or file == '..' then
+      goto continue
+    end
+    if not os.remove(path..'/'..file) then
```

Comment:
Oh, so `error()` works as a `print()`, but counts as error. I didn't mention it previously because I thought the whole test would fail immediately, which is not what we want, because we would like to  execute `after_each()` etc. But.. that all seems to work just fine.

I'll change that.